src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderEmploymentResponse.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel(description = "Offender Employment")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffenderEmploymentResponse {
+
+    @ApiModelProperty(value = "Offender booking id.", example = "14")
+    private Long bookingId;
```

Comment:
could all these be `final`?

Reply:
Not if this is constructed via a builder